services/messaging/src/main/java/org/eclipse/hono/telemetry/impl/ForwardingTelemetryDownstreamAdapter.java

```diff
@@ -47,12 +47,11 @@ public ForwardingTelemetryDownstreamAdapter(final Vertx vertx, final SenderFacto
     }
 
     protected void forwardMessage(final ProtonSender sender, final Message msg, final ProtonDelivery upstreamDelivery) {
-        ProtonHelper.accepted(upstreamDelivery, true);
         sender.send(msg);
     }
 
     @Override
     protected ProtonQoS getDownstreamQos() {
-        return ProtonQoS.AT_MOST_ONCE;
+        return ProtonQoS.AT_LEAST_ONCE;
```

Comment:
strikingly simple, isn't it? :+1:

Reply:
Yes, at the end a very slight change.. after a lot changes and tests before. But that is best. :)